src/spotbugs/spotbugs-excludes.xml

```diff
@@ -487,6 +487,7 @@
       <And>
         <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE"/>
         <Or>
+          <Class name="jenkins.fingerprints.FileFingerprintStorage"/>
```

Comment:
I would rather use `@SuppressFBWarnings` with targeted `justification`s.

Reply:
This is probably the same false positive as the other two, so all three can be triaged together in the future.